cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -1078,6 +1130,11 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     tabs[0].editors = [ReleaseDateEditor, GradingEditor, DueDateEditor];
                     tabs[1].editors = [ContentVisibilityEditor, ShowCorrectnessEditor];
 
+                    // If course is self paced and the custom PLS flag is active
```

Comment:
Nit: I'd say to remove this comment. I don't personally think it adds any information not captured in the code below it.